src/jsx.d.ts

```diff
@@ -24,20 +23,20 @@ type Booleanish = boolean | 'true' | 'false';
 
 /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ToggleEvent) */
 interface ToggleEvent extends Event {
-    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ToggleEvent/newState) */
-    readonly newState: string;
-    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ToggleEvent/oldState) */
-    readonly oldState: string;
+	/** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ToggleEvent/newState) */
+	readonly newState: string;
+	/** [MDN Reference](https://developer.mozilla.org/docs/Web/API/ToggleEvent/oldState) */
+	readonly oldState: string;
```

Comment:
I've reverted this in a few commits now, figure these formatting corrections are worth getting in just to get out of the way